src/index.js

```diff
@@ -0,0 +1,66 @@
+import './style.css';
+import dots from './images/dots.png';
+import CreateTask from './modules/createTask.js';
+
+// import bin from './images/bin.png';
+
+import enter from './images/enter.png';
+import refresh from './images/refresh.png';
+
+let arr = [];
+const enterButton = document.querySelector('.enter-images');
+enterButton.addEventListener('click', () => {
+  const str = document.querySelector('#place').value;
+  const number = arr.length;
+  const boolen = false;
+  const newTask = new CreateTask(str, boolen, number);
+  arr.push(newTask);
+  localStorage.setItem('arr', JSON.stringify(arr));
+  const texture = document.querySelector('#place');
+  texture.value = '';
+  window.location.reload();
```

Comment:
Kindly implement the function for editing task descriptions since it is part of the project requirements. :wink:

Reply:
- [ Optional] I can see that you forget to implement the editing function as required in the previous review , kindly fix this for your project to match all requirements :wink: